### ISSUE TITLE
CE(pseudo hard label, teacher_us)

### ISSUE THREAD

**Issue by miramirakim227** [kekmodel/MPL-pytorch#21]
<img width="694" alt="image" src="https://user-images.githubusercontent.com/85744300/129471726-635c3e17-fc55-4a1a-8cae-36ea28c005e0.png">
I have a small question about the above code. 
 
The aim of the cross entropy is to make the two different distributions be similar with each other. 
I understood the first CE loss but not the second CE loss, which confuses me a lot. 
In line 218, the latter argument of CE should follow the former argument.
In this case is the purpose of CE loss to make the raw output of teacher model to be sharpened?

**Comment by Xiao0728** [comment]
did you figure it out? and how was your implementation results? does it work? Thanks.
